core/modules/book/config/book.settings.json

```diff
@@ -4,5 +4,5 @@
     "book_child_type": "book",
     "book_navigation": 1,
     "book_navigation_options": ["tree", "pager"],
-    "book_links": ["book_add_child"]
+    "book_links": ["book_add_child", "book_reorder"]
```

Comment:
I guess, that's written directly - otherwise there would be newlines. Not sure if that matters (code standards?). It doesn't break anything, so it's no "real" problem.